registry/quilt_server/views.py

```diff
@@ -503,6 +506,20 @@ def blob_get(owner, blob_hash):
         put=_generate_presigned_url(S3_PUT_OBJECT, owner, blob_hash),
     )
 
+
+def download_object_preview(owner, obj_hash):
+    resp = s3_client.get_object(
```

Comment:
Can we guarantee that we're not reading more than MAX_PREVIEW_SIZE bytes? Maybe only fetch that limit from s3? Maybe add a some padding just in case the compressed file is bigger.